### PR TITLE
Track signup and login events

### LINKED ISSUE #616: Track of how many users sign up or log in
We want to know of how many users who download the app sign up or log in (or don't do anything at all).

### PR DESCRIPTION
**Pull request by simonrozsival**
Closes #616 

This is on hold until the issue is clarified.

### CODE REVIEW

**Comment by amulware** [comment]
I believe the issue has been clarified. Can work continue or are there more open questions?

**Comment by simonrozsival** [comment]
This PR is now ready for review.

File: Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs [hunk 1]
```diff
@@ -303,22 +316,31 @@ private void signUp()
             loginDisposable =
                 loginManager
                     .SignUp(Email, Password)
+                    .Do(_ => analyticsService.TrackSignUpEvent(AuthenticationMethod.EmailAndPassword))
                     .Subscribe(onDataSource, onError, onCompleted);
         }
 
         private async Task startPasswordManager()
         {
+            analyticsService.TrackPasswordManagerButtonClicked();
```
Comment (willsb) [inline Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs]: In principle, you shouldn't be able to click the button if it's not available unless there's a bug. Any reason why we are tracking this?

Comment (simonrozsival) [reply Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs]: Good point. I will remove it.

**Pull request merged**